src/lib.rs

```diff
@@ -181,6 +181,11 @@ pub mod storage;
 pub mod string;
 pub mod vec;
 
+// Workaround a compiler ICE in rust 1.83 to 1.86
```

Comment:
```suggestion
// FIXME: Workaround a compiler ICE in rust 1.83 to 1.86
```

Reply:
Can be done in a follow-up PR. Let's get this in before it conflicts with anything else.